src/main/java/edu/kis/vh/nursery/listAndArray/IntArrayStack.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.vh.nursery.listAndArray;
+
+public class IntArrayStack implements IntStackList {
+    private final int n = 12;
```

Comment:
Wrong name, written with a wrong convention.

Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?